modules/osd-aws-privatelink-about.adoc

```diff
@@ -2,6 +2,6 @@
 [id="osd-aws-privatelink-about.adoc_{context}"]
 = Understanding AWS PrivateLink
 
-A {product-title} cluster can be created without any requirements on public subnets, internet gateways, or network address translation (NAT) gateways. In this configuration, Red Hat uses AWS PrivateLink to manage and monitor a cluster in order to avoid all public ingress network traffic.
+A {product-title} cluster can be created without any requirements on public subnets, internet gateways, or network address translation (NAT) gateways. In this configuration, Red Hat uses AWS PrivateLink to manage and monitor a cluster to avoid all public ingress network traffic. Without a public subnet, it is not possible to configure an application router as public. Private application routers are the only option.
```

Comment:
Although the preceding sentence sets the context, the last sentence might be written better. For example, 
_Configuring private application routers is the only option._
or 
_Private application routers are the only option to ..._